packages/api-explorer/src/ApiExplorer.tsx

```diff
@@ -113,16 +113,24 @@ const ApiExplorer: FC<ApiExplorerProps> = ({
               value={{ searchSettings, setSearchSettings }}
             >
               <Page style={{ overflow: 'hidden' }}>
-                <Header
-                  specs={specs}
-                  spec={spec}
-                  specDispatch={specDispatch}
-                  toggleNavigation={toggleNavigation}
-                />
+                {envAdaptor.runAsEmbed() || (
```

Comment:
Is this a similar look to the standalone version? If standalone should use the same technique. Rather than using `runAsEmbed` change to headess. This should be a property as you only need to check it once and the property (a boolean can be passed in).